IPlayer/Player.cs

```diff
@@ -18,27 +22,49 @@ public void DisplayStatus()
 
     private void DisplayHp()
     {
-        if (this._didHpDecrease)
-        {
-            Console.ForegroundColor = ConsoleColor.Red;
-        }
-        else
+        switch (this.hpState)
         {
-            Console.ForegroundColor = ConsoleColor.Green;
+            case StateOfChange.Decreased:
+                Console.ForegroundColor = ConsoleColor.Red;
+                break;
+            case StateOfChange.Increased:
+                Console.ForegroundColor = ConsoleColor.Green;
+                break;
+            case StateOfChange.Same:
+                Console.ForegroundColor = ConsoleColor.White;
+                break;
+            default:
+                break;
```

Comment:
Take a look at switch expressions. This can be rewritten as:

```csharp
Console.ForegroundColor = direction this.hpState {
		StateOfChance.Increased => ConsoleColor.DarkGreen,
		StateOfChance.Decreased=> ConsoleColor.Red,
		_ => ConsoleColor.White
};
```